docs/x509/reference.rst

```diff
@@ -2542,6 +2543,17 @@ instances. The following common OIDs are available as constants.
 
         Corresponds to the dotted string ``"2.5.29.24"``.
 
+Helper Functions
+~~~~~~~~~~~~~~~~
+.. currentmodule:: cryptography.x509
+
+.. function:: random_serial_number()
+
+    .. versionadded:: 1.6
+
+    Generates a random serial number suitable for use when constructing
+    certificates or CRLs.
```

Comment:
Err, how do CRLs come into this? They reference a serial, but you don't generate a serial for them.